server/src/test/java/io/crate/auth/HostBasedAuthenticationTest.java

```diff
@@ -509,12 +523,15 @@ public void testKeyOrderIsRespectedInHbaConfig() {
             () -> "dummy"
         );
 
+        SSLSession sslSession = mock(SSLSession.class);
+        when(sslSession.getPeerCertificates()).thenReturn(new Certificate[]{mock(Certificate.class)});
+
```

Comment:
`cert` will never be matched without having a "proper payload". 
 See also change in `testClientCertAuthWithoutCert`